src/metabase/query_processor/streaming/xlsx.clj

```diff
@@ -392,25 +393,58 @@
         (str column-title " (" (currency-identifier col-viz-settings) ")")
         column-title))))
 
+(def ^:private auto-sizing-threshold
```

Comment:
One more suggestion actually... you should make this dynamic (e.g. `*auto-sizing-threshold*`) so we can change its value thread-locally with `binding` instead of `with-redefs`. That would mean we could make the `auto-sizing-threshold-test` parallelizable instead of having to single-thread it in the near-future when we flip test parallelization back on.